plugin.py

```diff
@@ -16,7 +17,7 @@ def plugin_unloaded() -> None:
 
 
 class LspPyrightPlugin(NpmClientHandler):
-    package_name = __package__
+    package_name = __package__.split(".")[0]
```

Comment:
In which case this is useful?

Reply:
Only if people place there plugin like: `Packages/MyPlugin/plugin/main.py`, then `__package__` would be `MyPlugin.plugin`. Useless in this project indeed.